### PR TITLE
Servings

### PR DESCRIPTION
**Pull request by subinakallyani**
change number of ingredient quantity according to servings

### CODE REVIEW

File: src/js/controller.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ searchContainer.addEventListener('submit', async e => {
   e.preventDefault();
   spinner(searchResults);
   await searchDataResult(searchBox.value);
-  console.log(e, 'sdf', state.search.results);
+  // console.log(e, 'sdf', state.search.results);
```
Comment (pranavmappoli) [inline src/js/controller.js]: remove

File: src/sass/_searchResults.scss [hunk 2]
```diff
@@ -40,3 +40,9 @@
     color: $color-grey-dark-2;
   }
 }
+.invisible {
+  display: none;
+}
+.visible {
+  display: block;
+}
```
Comment (pranavmappoli) [inline src/sass/_searchResults.scss]: remove if not required

File: src/views/recipeView.js [hunk 3]
```diff
@@ -3,7 +3,10 @@ import { Fraction } from 'fractional';
 import { renderError } from './spinner';
 console.log({ Fraction }, 'yyyy');
 const recipeContainer = document.querySelector('.recipe');
+
 export function renderRecipe(recipe) {
+  let servings = recipe.servings;
```
Comment (pranavmappoli) [inline src/views/recipeView.js]: why not const ?

File: src/views/recipeView.js [hunk 4]
```diff
@@ -104,7 +105,35 @@ ${recipe.ingredients
 </div>`;
   recipeContainer.innerHTML = '';
   recipeContainer.insertAdjacentHTML('afterbegin', html);
+  const btnIncrease = document.querySelector('.btn--increase-servings');
+  const btnDecrease = document.querySelector('.btn--decrease-servings');
+  btnIncrease.addEventListener('click', btnIncreaseHandler);
+  btnDecrease.addEventListener('click', btnDecreaseHandler);
+  btnDecrease.disabled = servings === 1 ? true : false;
```
Comment (pranavmappoli) [inline src/views/recipeView.js]: make it as a function, like initiateHandler or something

File: src/views/recipeView.js [hunk 5]
```diff
@@ -104,7 +105,35 @@ ${recipe.ingredients
...
+    recipe.servings = servings;
+    renderRecipe(recipe);
+  }
+  function newQuantity() {
+    recipe.ingredients.forEach(ing => {
+      ing.quantity = (ing.quantity * servings) / recipe.servings;
+    });
+    // console.log(servings, 'kjhg');
+    // console.log(recipe.ingredients, 'jhg');
+  }
```
Comment (pranavmappoli) [inline src/views/recipeView.js]: why u put this inside another function ? can move outside of main function ?

**Review by pranavmappoli — CHANGES_REQUESTED** [review]

File: src/js/controller.js (hunk 1 above)
Comment (subinakallyani) [reply src/js/controller.js]: removed

File: src/sass/_searchResults.scss (hunk 2 above)
Comment (subinakallyani) [reply src/sass/_searchResults.scss]: okay

File: src/views/recipeView.js (hunk 3 above)
Comment (subinakallyani) [reply src/views/recipeView.js]: declared the variable outside of the function using let

File: src/views/recipeView.js (hunk 4 above)
Comment (subinakallyani) [reply src/views/recipeView.js]: okay

File: src/views/recipeView.js (hunk 5 above)
Comment (subinakallyani) [reply src/views/recipeView.js]: okay

**Pull request merged**